builds/gyp/project.gyp

```diff
@@ -80,7 +80,6 @@
         '../../src/mlm_classes.h'
       ],
       'dependencies': [
-        '../../../libzmq/builds/gyp/project.gyp:libzmq',
```

Comment:
Please make sure to have libzmq cloned in the same parent directory as malamute and regenerate again - this shouldn't be removed